src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponentSpec.scala

```diff
@@ -415,7 +415,7 @@ class EntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers {
       withWorkspaceContext(workspace2) { context2 =>
         runAndWait(entityQuery.save(context2, x2))
         runAndWait(entityQuery.save(context2, x1))
-        val x2_updated = Entity("x2", "SampleSet", Map(AttributeName("library", "child") -> AttributeEntityReference("SampleSet", "x1")))
+        val x2_updated = Entity("x2", "SampleSet", Map(AttributeName.withDefaultNS("child") -> AttributeEntityReference("SampleSet", "x1")))
```

Comment:
Yes.  I recently sprinkled library in random locations - but now I'm switching them back because I don't need these tests to be library permissions tests